servers.json

```diff
@@ -930,5 +930,12 @@
         "addresses": [
             "aplaxy.net"
         ]
+    },
+    {
+        "id": "sefayovh",
+        "name": "real practice",
+        "addresses": [
+            "play.sefay.ovh"
```

Comment:
Please remove the `play.` prefix on the address.

You can take a look at the section [here](https://github.com/LunarClient/ServerMappings#ip-addresses) for more information on the addresses array.